agent/engine/docker_container_engine.go

```diff
@@ -47,6 +47,7 @@ const (
 // Timelimits for docker operations enforced above docker
 const (
 	pullImageTimeout        = 2 * time.Hour
+	loadImageTimeout        = 2 * time.Hour
```

Comment:
It's the same as the pull image timeout above. Since it's (conceptually) the same operation, I thought it made sense to have the same timeout.